cmdLine/src/main/scala/topgun/cmdline/ClassLoaderInfo.scala

```diff
@@ -0,0 +1,32 @@
+package topgun.cmdline
+
+
+import java.io.InputStream
+
+import org.objectweb.asm.ClassReader
+import org.objectweb.asm.tree.ClassNode
+
+import scala.collection.mutable
+
+class ClassLoaderInfo() {
+  var allClasses = new mutable.HashMap[String, ClassInfo]()
+
+  def lookup(clazzName: String, clazzLoader: ClassLoader): ClassInfo = {
+    val key = clazzName.replace(".", "/") + ".class"
```

Comment:
As this isn't always used it should not be calculated every time

so move this into the method below, and pass the classname as a parameter